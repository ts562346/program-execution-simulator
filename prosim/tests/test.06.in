test 3
LOOP 3
END
HALT

