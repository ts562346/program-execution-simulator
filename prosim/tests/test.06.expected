00000: HALT

DOOP count : 0
DOOP time  : 0
BLOCK count: 0
BLOCK time : 0
